automatic/kubernetes-cli/kubernetes-cli.nuspec

```
@@ -5,12 +5,12 @@
     <id>kubernetes-cli</id>
     <version>1.21.2</version>
     <packageSourceUrl>https://github.com/chocolatey-community/chocolatey-coreteampackages/tree/master/automatic/kubernetes-cli</packageSourceUrl>
-    <owners>chocolatey-community,Andrei Bejenaru</owners>
+    <owners>chocolatey-community,dgalbraith,Andrei Bejenaru</owners>
     <title>Kubernetes Command Line Interface (CLI)</title>
-    <authors>Google</authors>
+    <authors>Kubernetes Authors</authors>
     <projectUrl>https://kubernetes.io</projectUrl>
     <iconUrl>https://cdn.jsdelivr.net/gh/chocolatey-community/chocolatey-coreteampackages@5a8c40781821bbd2746754b7ce723938090bb7c6/icons/kubernetes-cli.png</iconUrl>
-    <copyright>(c) 2017 Kubernetes</copyright>
+    <copyright>(c) 2021 The Kubernetes Authors</copyright>
```

This should be:

```suggestion
    <copyright>&copy; 2021 The Kubernetes Authors</copyright>
```

Or

```suggestion
    <copyright><![CDATA[© 2021 The Kubernetes Authors]]></copyright>
```

The latter is most likely preferred in this case, as the former may not work as expected (not completely sure).